### PR TITLE
Fixed appveyor tests

### PR DESCRIPTION
**Pull request by wachterjohannes**
| Q | A
| --- | ---
| Bug fix? | yes
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | none
| Related issues/PRs | https://github.com/sulu/sulu/issues/2510
| License | MIT
| Documentation PR | none

#### What's in this PR?

This PR fixes the appveyor tests.

#### Why?

The appveyor tests should run before we can require it before merge

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -52,7 +52,8 @@
         "twig/twig": "~1.11",
         "zendframework/zend-stdlib": "~2.3",
         "zendframework/zendsearch": "@dev",
-        "doctrine/doctrine-fixtures-bundle": "2.2.*"
+        "doctrine/doctrine-fixtures-bundle": "2.2.*",
+        "phpdocumentor/reflection-docblock": "^2.0"
```
Comment (wachterjohannes) [inline composer.json]: it seems that appveyor (windows?) has problems with the version ^3.0 as we don't need it and 2.0 is also supported by the vendors this is ok for now.

File: tests/Resources/DataFixtures/Page/template_with_xinclude.xml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" ?>
 <template xmlns="http://schemas.sulu.io/template/template"
           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-          xmlns:xi="http://www.w3.org/2001/XInclude"
+          xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xI="http://www.w3.org/2001/XInclude"
```
Comment (danrot) [inline tests/Resources/DataFixtures/Page/template_with_xinclude.xml]: Is that really necessary?

**Comment by danrot** [comment]
The changes in the XML files should be reverted. This should be possible when we introduce an own XSD for properties, so that the file used for the xinclude can only use that schema, instead of the entire template. The properties xsd should then be imported into the template xsd.

File: tests/Resources/DataFixtures/Page/template_with_xinclude_properties.xml [hunk 3]
```diff
@@ -1,12 +1,10 @@
 <?xml version="1.0" ?>
 
-<template xmlns="http://schemas.sulu.io/template/template"
+<properties xmlns="http://schemas.sulu.io/template/template"
```
Comment (wachterjohannes) [inline tests/Resources/DataFixtures/Page/template_with_xinclude_properties.xml]: without any namespace it says that `properties` (without namespace) is not expected.

File: src/Sulu/Component/Content/Metadata/Loader/schema/template-1.0.xsd [hunk 4]
```diff
@@ -18,6 +18,7 @@
     </xs:attributeGroup>
 
     <xs:element name="template" type="templateType"/>
+    <xs:element name="properties" type="rootPropertiesType"/>
```
Comment (wachterjohannes) [inline src/Sulu/Component/Content/Metadata/Loader/schema/template-1.0.xsd]: this allows also properties as root node.

**Comment by danrot** [comment]
I've looked a little bit around, and it looks to me like the solution would be something like this: http://stackoverflow.com/a/6415076/1292378 

But I also couldn't get it running in a short amount of time. So if it is also ok for @chirimoya we could merge it like it is, although the `properties` tag should not be allowed to be the root element, and I think it is a serious BC break if we change the XSD files...

File: tests/Resources/DataFixtures/Page/template_with_xinclude.xml [hunk 5]
```diff
@@ -11,7 +11,9 @@
     <cacheLifetime>2400</cacheLifetime>
 
     <xi:include href='template_with_xinclude_main.xml'
```
Comment (chirimoya) [inline tests/Resources/DataFixtures/Page/template_with_xinclude.xml]: `href="template_with_xinclude_main.xml"`

**Comment by chirimoya** [comment]
ACK

**Pull request merged**